### PR TITLE
feat(layouts, layouts-ui): UX Identification of Library Components

### PR DESCRIPTION
**Pull request by VancheeZze**

### CODE REVIEW

File: packages/bodiless-layouts/src/ContentLibrary/ContentLibraryIndicator.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+const asDefaultLibraryItemIndicator = withDesign({
+  Wrapper: addClasses('hidden group-hover:flex bl-text-white bl-bg-gray-900 absolute bl-px-2 bl-py-1 md:-mt-5 md:-ml-5'),
+  Icon: asToken(
+    addClasses('material-icons bl-mr-2'),
+    addProps({ children: 'account_balance' }),
+  ),
+  Label: addProps({ children: 'Library Item' }),
+  InfoIcon: asToken(
+    addClasses('material-icons bl-ml-3'),
+    addProps({ children: 'info', href: '/___docs/', target: '_blank' }),
```
Comment (misterjones) [inline packages/bodiless-layouts/src/ContentLibrary/ContentLibraryIndicator.tsx:71]: Once documentation is approved (#1245), I think the ideal location for this link to lead would be: [Flow Container : Saving a component in the Content Library](https://johnsonandjohnson.github.io/Bodiless-JS/#/Components/FlowContainer?id=saving-a-component-in-the-content-library).

```suggestion
    addProps({ children: 'info', href: '/___docs/#/Components/FlowContainer?id=saving-a-component-in-the-content-library', target: '_blank' }),
```

**Comment by TatsianaMiraniuk** [comment]
**issue_1**:

the label shows behind of the following components:
- Accordion
- RTE
- Cards (regular and contentful)
- YouTube
- Carousel
Screen Shot 2021-11-23 at 6 49 02 PM

Screen Shot 2021-11-23 at 6 49 18 PM

Screen Shot 2021-11-23 at 6 48 56 PM

Screen Shot 2021-11-23 at 6 49 07 PM


Screen Shot 2021-11-23 at 6 51 13 PM

**Comment by TatsianaMiraniuk** [comment]
@rageshchalil ,

issue#2:
the following AC is not met fully:
'AC. The information icon includes a link to the documentation for working with the library items that opens in a new tab'
Home page of the Docs opens, not the page about Library items.

File: packages/bodiless-layouts/src/ContentLibrary/ContentLibraryIndicator.tsx (hunk 1 above)
Comment (marcopagliarulo) [reply packages/bodiless-layouts/src/ContentLibrary/ContentLibraryIndicator.tsx:71]: @misterjones probably the link to http://localhost:8005/___docs/#/ContentEditorUserGuide/?id=content-library is a better location, then it contains the link to the location suggested by you

Comment (misterjones) [reply packages/bodiless-layouts/src/ContentLibrary/ContentLibraryIndicator.tsx:71]: @marcopagliarulo, agreed — good call.

Updated suggestion. 👍

**Review by TatsianaMiraniuk — APPROVED** [review]

**Review by marcopagliarulo — APPROVED** [review]

**Pull request merged**